try.html

```diff
@@ -889,6 +889,10 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='/amo/stars/dustman.svg' alt=''/></td>
     <td><code>https://img.shields.io/amo/stars/dustman.svg</code></td>
   </tr>
+  <tr><th> Swagger Validator: </th>
+    <td><img src='/swagger/v/2.0/http/petstore.swagger.io/v2/swagger.json.svg' alt=''/></td>
```

Comment:
This example is a little difficult to follow because it has `swagger` in it three times. Could you look for another, popular API to use as an example?